llvm/lib/Target/AMDGPU/AMDGPUAttributor.cpp

```diff
@@ -439,6 +451,36 @@ struct AAAMDAttributesFunction : public AAAMDAttributes {
       indicatePessimisticFixpoint();
       return;
     }
+
+    { // FLAT_SCRATCH_INIT
```

Comment:
This is misuse of `initialize`. This code should go to `updateImpl` of the corresponding AA.